### PR TITLE
remove dynamic pressure related diagnostics

### PR DESCRIPTION
**Pull request by yairchn**
We have a branch storing these changes 

https://github.com/CliMA/pycles/tree/yc/master_pressure_diagnostics

and we can make a PR in the future with nameless flags to call these pressure related diagnostics. For now lets clean up the code for the LES - GCM runs

### CODE REVIEW

File: Tracers.pyx [hunk 1]
```diff
@@ -407,7 +400,6 @@ cdef class UpdraftTracers:
             double [:] w_half = np.zeros((Gr.dims.npg),dtype=np.double, order='c')
             double [:] dpalphadz = np.zeros((Gr.dims.npg),dtype=np.double, order='c')
             double [:] wpz_half = np.zeros((Gr.dims.npg),dtype=np.double, order='c')
```
Comment (szy21) [inline Tracers.pyx]:
```suggestion

```

File: Tracers.pyx [hunk 2]
```diff
@@ -481,9 +471,6 @@ cdef class UpdraftTracers:
         NS.write_profile('updraft_perturbation_pressure_potential', tmp[Gr.dims.gw:-Gr.dims.gw], Pa)
 
         tmp = Pa.HorizontalMeanConditional(Gr, &wpz_half[0], &self.updraft_indicator[0])
```
Comment (szy21) [inline Tracers.pyx]:
```suggestion

```

**Review by szy21 — APPROVED** [review]
I think there are two more lines we can remove (see the comments I left), otherwise it looks good to me, thanks!

File: TimeStepping.pyx [hunk 3]
```diff
@@ -67,12 +67,6 @@ cdef class TimeStepping:
         except:
             self.acceleration_factor = 1.0
 
-        try:
-            self.statIOdt = namelist['stats_io']['frequency']
-        except:
-            Pa.root_print('statsIOfrequency set to dt_max')
-            self.statIOdt = self.dt_max
```
Comment (jiahe23) [inline TimeStepping.pyx]: This is not related to the pressure diagnostic output... Not sure if we could/should delete it

File: TimeStepping.pyx [hunk 4]
```diff
@@ -470,14 +285,15 @@ cdef class TimeStepping:
                             w = fmax(fabs( DV.values[DV.sedv_index[isedv]*Gr.dims.npg + ijk ] + PV.values[w_shift+ijk]), w)
 
                         cfl_max_local = fmax(cfl_max_local, self.dt * (fabs(PV.values[u_shift + ijk])*dxi[0] + fabs(PV.values[v_shift+ijk])*dxi[1] + w*(1.0/Gr.dzpl[k])))
+                        #cfl_max_local = fmax(cfl_max_local, self.dt * (fabs(PV.values[u_shift + ijk])*dxi[0] + fabs(PV.values[v_shift+ijk])*dxi[1] + w*dxi[2]))
```
Comment (jiahe23) [inline TimeStepping.pyx]: what is this commented line for?
